rpc-client/src/nonblocking/rpc_client.rs

```diff
@@ -2150,6 +2150,10 @@ impl RpcClient {
     /// # })?;
     /// # Ok::<(), Error>(())
     /// ```
+    #[deprecated(
+        since = "1.18.18",
+        note = "Do not use; getStakeActivation is deprecated on the JSON-RPC server"
```

Comment:
i probably should've caught this in the master PR, but is there guidance to be offered here? `rpc_client.get_account()?.get_stake_activation()` or w/e

Reply:
Unfortunately, the replacement approach isn't very simple. It's essentially:
- Get StakeHistory account
- Get epoch
- Use Delegation from stake account to call `Delegation::stake_activating_and_deactivating(epoch, &stake_history, None)`

I'll try to think of a simple way to point people in this direction and PR separately.